WordPress/Classes/ViewRelated/SuggestionsListViewModel.swift

```diff
@@ -123,15 +123,35 @@ import CoreData
         }
         switch suggestions {
         case .users(let userSuggestions):
+            let sortedUserSuggestions = Self.sort(
+                userSuggestions: userSuggestions, by: searchQuery
+            )
             return Self.searchResultByMovingProminentSuggestionsToTop(
-                userSuggestions: userSuggestions,
+                userSuggestions: sortedUserSuggestions,
                 prominentSuggestionsIds: prominentSuggestionsIds
             )
         case .sites(let siteSuggestions):
             return .sites(siteSuggestions)
         }
     }
 
+    private static func sort(userSuggestions: [UserSuggestion], by searchQuery: String) -> [UserSuggestion] {
+        guard !searchQuery.isEmpty, !userSuggestions.isEmpty else { return userSuggestions }
+        var prefixedUserSuggestions = [UserSuggestion]()
+        var otherUserSuggestions = [UserSuggestion]()
+
+        userSuggestions.forEach { suggestion in
+            guard let displayName = suggestion.displayName else { return }
```

Comment:
This is a corner case, but if the user doesn't have a display name for some reason, they will be filtered out completely. I think we should add them to `otherUserSuggestions` in that case. Wdyt?

Reply:
Yes, that is a good idea. I going to apply it.